homework/src/index.js

```diff
@@ -30,18 +42,141 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { html: err.message, class: 'alert-error' });
-      } else {
-        createAndAppend('pre', root, { html: JSON.stringify(data, null, 2) });
-      }
+  //this is an object creator
+  function RepositoryInfo(sourceObj) {
+    this.Description = sourceObj.description;
+    this.Forks = sourceObj.forks;
+    const dateOfUpdate = new Date(sourceObj.updated_at);
+    this.Updated = dateOfUpdate.toLocaleString();
+  }
+
+  //info-box of the page
+  function createInfoBox(repoData, container) {
+
+    container.innerHTML = "";
+
+    const info = new RepositoryInfo(repoData);
```

Comment:
I'm not sure whether creating on object here rather than using the repo data directly adds value. Perhaps this was just a finger exercise?